icc-api/model/CalendarItem.ts

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
This is a regression as only the CalendarItemDto should be exposed in the icc-api. The file should be deleted (new genloc required)